2020/26xxx/CVE-2020-26990.json

```diff
@@ -19,27 +19,7 @@
                                 "version": {
```

Comment:
('Expecting property name enclosed in double quotes: line 78 column 13 (char 3025)',)